packages/dashboard/src/components/negotiations-panel.tsx

```diff
@@ -275,17 +318,74 @@ export const NegotiationsPanel = React.memo((props: NegotiationsPanelProps) => {
     }
 
     updateNegotiationTrajectory();
+    setSelected(nodeIds);
+  };
+
+  const handleClearAllCurrNegotiations = () => {
+    localStorage.setItem('conflictIds', Object.keys(conflicts).toString());
+    setNegotiationContents({});
+  };
+
+  const handleResetNegotiations = () => {
+    setExpanded([]);
+    setSelected('');
+    setNegotiationTrajStore({});
+  };
+
+  const handleRestoreNegotiations = () => {
+    localStorage.setItem('conflictIds', '');
+    setNegotiationContents({});
+  };
+
+  const handleNodeToggle = (event: React.ChangeEvent<{}>, nodeIds: string[]) => {
+    setExpanded(nodeIds);
   };
 
   return (
     <Typography variant="body1" component={'span'}>
+      <div style={{ padding: '0.5rem 1rem' }}>
```

Comment:
What do you think about using className here and move it to `useStyles` to keep consistency?